src/components/Main/Containment/ContainmentGraph.tsx

```diff
@@ -21,9 +21,7 @@ function draw({ data, width, height, onDataChange, d3ref }: DrawParams) {
   if (d3ref.current === null) {
     return
   }
-  const newData = data.map(d => {
-    return { t: d.t, y: d.y }
-  }) // copy
+  const newData = data.map((d): TimePoint => ({ t: d.t, y: d.y })) // copy
```

Comment:
This function is now a one-liner instead of a full function with a return statement.